worlds/noita/__init__.py

```diff
@@ -35,6 +35,10 @@ class NoitaWorld(World):
 
     web = NoitaWeb()
 
+    def generate_early(self):
```

Comment:
> How was this tested?

I don't think this is N/A with this. What's the outcome when you use unsupported characters in the slot name?